### ISSUE TITLE
Request: Bulk update of query param options.

### ISSUE THREAD

**Issue by brianmcmichael** [johngodley/redirection#1378]
Hi!

Thank you for providing the option to ignore query params in the 4.0 release! Facebook is really messing with things by adding those query params.

I have several hundred redirect urls I'd like to update to ignore query params. After changing the setting in options it looks like new urls are respecting this setting, but the existing links need to be manually. It would be nice to have the ability to bulk update these links in the dropdown.

Thanks!

**Comment by johngodley** [comment]
A bulk editing feature is planned for the future, but you should be able to set defaults from the options page anyway.

**Comment by johngodley** [comment]
Note that 4.0 alpha 2 fixes a bug with default query param settings.

**Comment by brotons** [comment]
@johngodley  there is still no bulk edit feature in  Redirection-plugin to move to a GROUP ?

**Comment by johngodley** [comment]
There is not. This issue will be closed when there is.

**Comment by brotons** [comment]
why will it be closed? how difficult after many links that want to be grouped into groups? Should I do the editing one by one?

grouping many links into groups one by one, it is a time-consuming work.

**Comment by johngodley** [comment]
This issue will be closed when the feature is complete.

**Comment by brotons** [comment]
thanks. Hopefully it will be finished quickly because millions of users of this plugin are needed.

**Comment by galbaras** [comment]
+1. Very useful feature to have

**Comment by ShadowXVII** [comment]
Would be a very useful feature. Currently have to update hundreds of rules one by one.

- Query Parameters
- HTTP Code
- Group

**Comment by jwest75674** [comment]
Other issues related issues (issues which will have an adequate solutions once the bulk editing feature is complete):
#3016
#1933 
#1378
#3116 
#3154 
#2554
